### ISSUE TITLE
Glyphicons must be removed

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/cas#272]
**[Original report](https://bitbucket.org/laborstudents/cas-flask/issue/272) by Anne Achieng (Bitbucket: [Otianne](https://bitbucket.org/Otianne), GitHub: [OtienoAnne](https://github.com/OtienoAnne)).**

----------------------------------------

The NVDA cannot read the glyphicons used for indicating the accessibility levels in the room information form. This is against the accessibility criteria.

files affected: RoomPrefrences HTML and JavaScript
On branch jamal_basanta.

**Comment by bryantal** [comment]
**Original comment by Evan Johnson (Bitbucket: [johnsone](https://bitbucket.org/johnsone), GitHub: [johnsonwe](https://github.com/johnsonwe)).**

----------------------------------------

Branch: glyphicons_removal

**Comment by bryantal** [comment]
**Original comment by Evan Johnson (Bitbucket: [johnsone](https://bitbucket.org/johnsone), GitHub: [johnsonwe](https://github.com/johnsonwe)).**

----------------------------------------

Kenny has done some work on this issue since I've last worked on branch: AccessibilityRP. Merging in AccessibilityRP.

**Comment by bryantal** [comment]
**Original comment by Evan Johnson (Bitbucket: [johnsone](https://bitbucket.org/johnsone), GitHub: [johnsonwe](https://github.com/johnsonwe)).**

----------------------------------------

I believe that all the glyphicons have now been removed from the system, I'm going to review this.

**Comment by bryantal** [comment]
**Original comment by Evan Johnson (Bitbucket: [johnsone](https://bitbucket.org/johnsone), GitHub: [johnsonwe](https://github.com/johnsonwe)).**

----------------------------------------

Fix awaiting approval
